scripts/cmdlet.sh

```diff
@@ -24,20 +24,20 @@ else
 fi
 
```

Comment:
issues:
1. Use $(...) notation instead of legacy backticked `...`.
2. Double quote to prevent globbing and word splitting.